Update docs and validation to account for dynamic implementation (#129)
AlaskaAirlines/auro-radio: README.md

<!-- The below code snippet is automatically added from ./../apiExamples/basic.html -->

```html

> <auro-radio label="Yes" name="radioDemo" value="yes"></auro-radio>

Reviewer: `id` is a required part of the html5 spec for all form elements. We have to include it here and have it passed down to the inner `input` element.

The `<auro-radio>` element is designed to be a single component for the use of creating an input `type="radio"` with an associated `<label>` that meets all use case and accessibility standards.

The following examples illustrate fully functional `<auro-radio>` elements wrapped with the `<auro-radio-group>` element. The `<auro-radio-group>` element is REQUIRED in order to mantain the relationship between individual `<auro-radio>` elements for functional radio button actions.

> **Note**: The `<auro-radio>` element is not intended to be a standalone element. It should always be used in conjunction with other `<auro-radio>` elements within the `<auro-radio-group>` element.

Reviewer: ```suggestion
**Note**: The `<auro-radio>` element is only for to for use as a set of two or more `<auro-radio>` elements within an `<auro-radio-group>` element.
```

<!-- The below code snippet is automatically added from ./../apiExamples/basic.html -->

```html

> <auro-radio label="Yes" name="radioDemo" value="yes"></auro-radio>

Reviewer: See above comment about `id` requirement.
